Compile/interface/Filters/ZJetCutsFilter.h

```diff
@@ -472,7 +634,24 @@ class ValidZCut : public ZJetFilterBase
                        ZJetProduct const& product,
                        ZJetSettings const& settings) const override
     {
-        return (product.m_zValid && std::abs(product.m_z.p4.M()-ZMass)<ZMassRange);
+        LOG(DEBUG) << "\n[" << this->GetFilterId() << "]";
+        LOG(DEBUG) << "Accepted Z mass: " << ZMass << " (pdg) +- " << settings.GetZMassRange()
```

Comment:
Is Zmass fixed? Or is it a config key one can change?

Reply:
Config key

Reply:
Then I would remove the pdg string